src/components/Header/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Link, BrowserRouter as Router } from 'react-router-dom';
+
+const Header = () => (
+  <header>
```

Comment:
If this header is not needed as an element in the DOM or doesn't improve the structure, we can replace it with [Fragments](https://reactjs.org/docs/fragments.html)

Reply:
<header />
<nav />
<main />
<footer />

all these tags are useful for SEO purposes, therefore I found it necessary to have `<header />`.

Reply:
Makes sense 👌